tot/models/models_simple.py

```diff
@@ -204,11 +250,15 @@ def predict(self, df: pd.DataFrame, df_historic: pd.DataFrame = None):
         _check_min_df_len(df=df, min_len=self.n_forecasts)
         if df_historic is not None:
             df = self.maybe_extend_df(df_historic, df)
-        df, received_ID_col, received_single_time_series, _ = prep_or_copy_df(df)
         fcst_df = _predict_darts_model(
-            df=df, model=self, n_req_past_obs=self.n_lags, n_req_future_obs=self.n_forecasts, retrain=False
+            df=df,
+            model=self,
+            n_req_past_obs=self.n_lags,
+            n_req_future_obs=self.n_forecasts,
```

Comment:
I assume `obs` are `observations`? being explicit and verbose makes it easier to read for people less familiar with the code